app/components/agenda/agendaitem/agendaitem-postponed.js

```diff
@@ -0,0 +1,97 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import addWeeks from 'date-fns/addWeeks';
+import { PAGE_SIZE } from 'frontend-kaleidos/config/config';
+
+/**
+ * @argument subcase
+ * @argument meeting
+ * @argument agendaActivity
+ */
+export default class AgendaitemPostponed extends Component {
+  @service store;
+  @service agendaService;
+
+  @tracked canPropose = false;
+  @tracked modelsForProposedAgenda;
+  @tracked newMeeting;
+
+  constructor() {
+    super(...arguments);
+    this.loadProposedStatus.perform();
+    this.loadProposableMeetings.perform();
+  }
+
+  @task
+  *loadProposedStatus() {
+    // We have to check if the proposed agendaitem is already proposed for a new meeting before showing the button
+    // we have to generate a link to the latest meeting (not the next if multiple).
+    // filtering on meeting.plannedStart will not find any if reProposing happened before the meeting "started" at 10 am
```

Comment:
This comment confuses me more than it clarifies. Is this here to explain the filter on `this.args.agendaActivity.startDate.toISOString()`? 
In that case, I would just put `Filtering on agenda-activities that are more recent than the agenda-activity of the postponed agendaitem`